docs/platforms/flutter/integrations/slow-and-frozen-frames-instrumentation.mdx

```diff
@@ -9,20 +9,21 @@ categories:
 ---
 
 Unresponsive UI and animation hitches annoy users and degrade the user experience.
-Two measurements to track these types of experiences are **slow frames** and **frozen frames**.
-Sentry's slow and frozen frames instrumentation helps track these measurements.
+Three measurements to track these types of experiences are **slow frames**, **frozen frames** and **frames delay**
+Sentry's slow and frozen frames and frames delay instrumentation helps track these measurements.
```

Comment:
```suggestion
Use this integration to find out about these problems in your app by tracking and showing you **slow frames**, **frozen frames** and **frames delay** metrics for spans.
```

Reply:
@kahest hope you don't mind a made a few tweaks to your suggestion :)